docs/versions.mdx

```diff
@@ -68,6 +59,25 @@ You can rename any version to something other than the version’s timestamp.
 
 The version’s timestamp will still be shown under the version name.
 
+## Auto-saved versions
+
+There are some cases where versions will automatically be created for a machine Auto-saved versions are indicated in the list of versions with the Stately bot icon.
+
+### Initial versions
+
+An initial version is automatically saved for new machines when:
+
+- you fork a machine from an existing machine or project
+- you duplicate an existing machine in its project
+
+These versions are labeled “Initial version”.
+
+### Periodic snapshot versions
+
+To reduce the likelihood of data loss, versions are automatically saved hourly as long as edits have been made to the project machine. These versions are labeled with a timestamp that can be edited by you. They work just like other versions that you can view, restore, or delete.
```

Comment:
```suggestion
To reduce the likelihood of data loss, versions are automatically saved hourly (approx.) as long as edits have been made to the project machine. These versions are labeled with a timestamp, and these labels can be edited. They work just like other versions that you can view, restore, or delete.
```

Didn't want to imply that the timestamps can be edited 😨 Also didn't want to guarantee hourly (it may not show up as exactly every hour, depending on edits, closing window, etc.)

Reply:
These are good distinctions to make! Thansk.